demo/locales/en.json

```diff
@@ -50,6 +50,7 @@
   "DEMO_DELETE_STORED_PROMPT": "Delete the offline copy?",
   "DEMO_DEMO_MODE_HEADER": "DEMO MODE",
   "DEMO_DISABLE_AUDIO": "Disable Audio",
+  "DEMO_DISABLE_IMAGE": "Disable Image",
```

Comment:
Is "Image" here only applicable to thumbnails or something else too?
If it's only for thumbnails, let's go ahead and name it "Disable thumbnails" :)

Reply:
OK, I'll change it!

Reply:
Done!